<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_detailAndCheck</class>
 <widget class="QDialog" name="admin_detailAndCheck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView_display">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>531</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_check">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>370</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>审核通过</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>370</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>当前状态：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_checking">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>正在查看：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_uncheck">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>370</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>审核不通过</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_suspend">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>370</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>缺货挂起</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
